frontends/election-manager/src/components/smartcard_modal/card_details_view.tsx

```diff
@@ -103,70 +111,72 @@ export function CardDetailsView({
-      {/* An empty div to maintain space between the header and subsequent p. TODO: Consider adding
-        a `maintainSpaceBelowHeaders` prop to `Prose` */}
-      <div />
+    <Prose textCenter theme={fontSizeTheme.medium}>
+      <h1>{userRoleToReadableString(role)} Card</h1>
+      {role !== 'superadmin' && <p>{electionDisplayString}</p>}
+
+      <HorizontalRule />
       {actionStatus && (
-        <StatusMessage
-          actionStatus={actionStatus}
-          programmedUser={programmedUser}
-        />
+        <StatusMessageContainer large={actionStatus.status === 'Success'}>
+          <StatusMessage
+            actionStatus={actionStatus}
+            programmedUser={programmedUser}
+          />
+        </StatusMessageContainer>
       )}
-      <CardDetailsTable>
-        <thead>
-          <tr>
-            <th>Role</th>
-            <th>Election</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <td>{userRoleToReadableString(role)}</td>
-            <td>{electionDisplayString}</td>
-          </tr>
-        </tbody>
-      </CardDetailsTable>
-      <p>Remove card to leave this screen.</p>
-      {'passcode' in programmedUser &&
-        (role === 'superadmin' ||
-          // If the card is from a prior election, no need to display PIN resetting. Unprogramming is
-          // the only meaningful action in this case
-          doesCardElectionHashMatchMachineElectionHash) && (
-          <p>
-            <Button disabled={cardJustProgrammed} onPress={resetCardPin}>
-              Reset Card PIN
-            </Button>
-          </p>
-        )}
-      {/* Don't allow unprogramming super admin cards to ensure election officials don't get
-        accidentally locked out. Likewise prevent unprogramming when there's no election definition
-        on the machine since cards can't be programmed in this state */}
-      {(role === 'admin' || role === 'pollworker') && electionDefinition && (
-        <React.Fragment>
-          <p>
+      {!showingSuccessOrErrorMessage && possibleActions.size > 0 && (
+        <p>
+          {possibleActions.has('PinReset') && (
+            <Button onPress={resetCardPin}>Reset Card PIN</Button>
+          )}
+          {possibleActions.size > 1 && ' '}
+          {possibleActions.has('Unprogram') && (
             <Button
               danger={doesCardElectionHashMatchMachineElectionHash}
               onPress={unprogramCard}
               primary={!doesCardElectionHashMatchMachineElectionHash}
             >
               Unprogram Card
             </Button>
-          </p>
-        </React.Fragment>
+          )}
+        </p>
+      )}
+      {(actionStatus || possibleActions.size > 0) && <HorizontalRule />}
+
+      {showingSuccessOrErrorMessage ? (
+        <Text bold>Remove card to continue.</Text>
+      ) : (
+        <p>
+          Remove card to{' '}
+          {possibleActions.size > 0 ? 'cancel' : 'leave this screen'}.
```

Comment:
When we do translation/internationalization it is necessary to have full sentences to translate as sentence structure changes in different languages. This is why my commit had separated these into two distinct phrases rather than one phrase with conditional logic inside it. 

If I'm doing a string search for a phase that may in a test, it's also nice to not have to think about what part of the sentence may be a conditional, and just be able to search for the full phrase.

Reply:
Ah yes I know you've pointed this out before! Will update and keep in mind for future changes